public/js/signup.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Alert the user through a message if the USN is avaialble or not 
+ */
+$('#usn').change(() => {
+    var data = { usn : $('#usn').val() };
+    var url = "/existUSN"; 
+    // Function which displays whether the USN is avaialable for use or not
+    displayMessage = (res) => {
+        $("#usn_response").show();
+        if(res.exist) {
+            $("#usn_response").html("<span class='not-exists'>* USN Already in use.</span>");       
+        }
+        else {
+            $("#usn_response").html("<span class='exists'>Available.</span>");
```

Comment:
Does this appear while typing the USN?

Reply:
Not while typing USN. But after focusing outside the USN input